doc-content/enterprise-only/installation/installer-run-cli-proc.adoc

```diff
@@ -9,6 +9,7 @@ For security reasons, you should run the installer as a non-root user.
 ====
 
 .Prerequisites
+* A supported JDK is installed. See https://access.redhat.com/articles/3405381[Red Hat Process Automation Manager 7 Supported Configurations] for a list of supported JDKs.
```

Comment:
Suggest, per optimal IBM linking strategy, "For a list of supported JDKs, see ...".